### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by lopezfra**
Create new list view option to lastest movies and latest episodes in dashboard

### CODE REVIEW

File: interfaces/default/js/dash.js [hunk 1]
```diff
@@ -172,6 +172,33 @@ function loadRecentMovies() {
...
+        $('<tr>').append($('<td>').html('&nbsp;').attr('colspan', '2')),
+        $('<tr>').append($('<td>').html('&nbsp;').attr('colspan', '2')),
+        $('<tr>').append($('<td>').html('&nbsp;').attr('colspan', '2')),
+        $('<tr>').append($('<td>').html('&nbsp;').attr('colspan', '2'))
+      )
+      return
+    }
+    $.each(result.movies, function(i, item) {
+
+      var fanart = WEBDIR + 'kodi/GetThumb?h=240&w=430&thumb=' + encodeURIComponent(item.fanart)
```
Comment (Hellowlol) [inline interfaces/default/js/dash.js]: This seem to be unused. Please remove it. I'll test and merge your pr later
today. Thanks for your contribution.

Comment (lopezfra) [inline interfaces/default/js/dash.js]: Hi! I use in https://github.com/LopezRegaladoFj/HTPC-Manager/blob/37af2abb3efbae7c8b4f5aaed7b6fad36d2dbfa2/interfaces/default/html/dash.html lines 314,315,316

Comment (lopezfra) [inline interfaces/default/js/dash.js]: Ah,ok,yo talk about 'var fanart', ok,i am going to remove

Comment (Hellowlol) [inline interfaces/default/js/dash.js]: Yes i was referring to the "var fanart."

Comment (lopezfra) [inline interfaces/default/js/dash.js]: commited

**Comment by Hellowlol** [comment]
Thanks! Can you also squash the commits?

**Comment by lopezfra** [comment]
done!

**Comment by Hellowlol** [comment]
Thanks!

**Comment by Hellowlol** [comment]
Thanks for your contribution! :) I did some small changes to your pr as i wanted to showname to be on the left like it is with the other "dash modules" and fixed a copy paste error.

**Pull request merged**